app/views/find/v2/results/index.html.erb

```diff
@@ -114,6 +114,7 @@
             <%= render Courses::SummaryCardComponent.new(
               course:,
               location: @search_params[:formatted_address] || @search_params[:location],
+              distance_from_location: @distance_from_location,
```

Comment:
Because is not set, `@distance_from_location` is nil all the time making `0 miles from #{location}` always appear.

For the results the distance is already on the SQL results so maybe `distance_from_location` is not needed specific here?